src/content/_index.en.md

```diff
@@ -19,7 +19,37 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
 * [subctl](./deployment/), a friendly deployment tool 
 * Support for interconnecting clusters with [overlapping CIDRs](./architecture/globalnet/)
 
+<style>
+.mygrid {  
+    display: grid;
+    grid-gap: 12px;  
+    grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
+    grid-template-rows: repeat(2, 100px);  
+    padding-bottom: 2em;
```

Comment:
I think we should add a bit more space. With padding-bottom set to 2cm it looks better to me (and also uncovers the h5 titles).

Reply:
I'm not sure about what 2cm means in css :-) em I think is 16px, cm may be depends on dpi? :) I remember there were some of those which were not safe to use.. but that's all I know , can I try with more px or ems? probably Tuomas can better know about this.

Reply:
Yeah.. wasn't sure either :)
https://www.w3schools.com/cssref/pr_padding-bottom.asp

5em is the equivalent of 2cm and also looks good to me.